src/neo/SmartContract/InteropService.Contract.cs

```diff
@@ -166,6 +172,14 @@ private static bool Contract_IsStandard(ApplicationEngine engine)
                 engine.CurrentContext.EvaluationStack.Push(isStandard);
                 return true;
             }
+
+            private static bool Contract_CreateStandardAccount(ApplicationEngine engine)
+            {
+                if (!engine.TryPop(out ReadOnlySpan<byte> pubKey)) return false;
+                UInt160 scriptHash = SmartContract.Contract.CreateSignatureRedeemScript(ECPoint.DecodePoint(pubKey, ECCurve.Secp256r1)).ToScriptHash();
+                engine.CurrentContext.EvaluationStack.Push(scriptHash.ToArray());
```

Comment:
Please, use only `Push` like https://github.com/neo-project/neo/pull/1511

Reply:
Thanks for reminding, have modified correspondingly